sdk/azidentity/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 0.13.1 (Unreleased)
 
 ### Features Added
+* Force pipelines
```

Comment:
Can you expand on this? (I have no idea what it means)

Reply:
I'm just touching azidentity to force the identity pipelines to run. I want to make sure I don't incidentally break your pipelines

Reply:
Ah, I see. You can also trigger arbitrary pipelines by commenting e.g. "/azp run go - azidentity"